pkgs/top-level/all-packages.nix

```diff
@@ -34254,7 +34254,9 @@ with pkgs;
 
   planetary_annihilation = callPackage ../games/planetaryannihilation { };
 
-  prismlauncher = libsForQt5.callPackage ../games/prismlauncher { };
+  prismlauncher-qt5 = libsForQt5.callPackage ../games/prismlauncher { };
```

Comment:
Why do we need that if the default version used by upstream is qt6?

Reply:
At least for now it's useful if we get any bug reports to be able to confirm easily if it applies on the qt5 version too.

Personally I'm not the most sure about the quality of the qt6 packaging (this is probably because I worked on it lol) so maybe that's just a lack of confidence about my own contributions!

Happy to remove it or keep it, whatever the consensus is.

Reply:
The reason why upstream maintains Qt 5 support, is that some users on Linux do not have theming engines capable of Qt 6. The KDE project has yet to release KDE Frameworks 6 which is why most Plasma and LXQt users still prefer to use Qt 5 apps over Qt 6 ones.

Reply:
@SuperSandro2000 any more input given these replies?